contracts/DCAFeeManager/DCAFeeManager.sol

```diff
@@ -121,11 +133,11 @@ contract DCAFeeManager is RunSwap, AccessControl, Multicall, IDCAFeeManager {
   /// @inheritdoc IDCAFeeManager
   function availableBalances(IDCAHub _hub, address[] calldata _tokens) external view returns (AvailableBalance[] memory _balances) {
     _balances = new AvailableBalance[](_tokens.length);
-    for (uint256 i; i < _tokens.length; i++) {
+    for (uint256 i = 0; i < _tokens.length; i++) {
```

Comment:
Why not use unchecked in these?

Reply:
Because `availableBalances` is meant to be used off-chain